compose-files/aquarius.yml

```diff
@@ -18,3 +20,15 @@ services:
       DB_CA_CERTS: ${DB_CA_CERTS}
       DB_CLIENT_KEY: ${DB_CLIENT_KEY}
       DB_CLIENT_CERT: ${DB_CLIENT_CERT}
+      LOG_LEVEL: DEBUG
+      EVENTS_ECIES_PRIVATE_KEY: "0x5d75837394b078ce97bc289fa8d75e21000573520bfa7784a9d28ccaae602bf8"
+      ARTIFACTS_PATH: "/ocean-contracts/artifacts"
+      ADDRESS_FILE: ${OCEAN_ADDRESS_FILE:-/ocean-contracts/artifacts/address.json}
```

Comment:
`OCEAN_ADDRESS_FILE` only exists here. Is this supposed to be `ADDRESS_FILE`, the one from the main script?